src/site/stages/build/process-cms-exports/transformers/node-leadership_listing.js

```diff
@@ -14,7 +14,9 @@ const transform = (entity, { ancestors }) => ({
   fieldAdministration: entity.fieldAdministration[0],
   fieldDescription: getDrupalValue(entity.fieldDescription),
   fieldIntroText: getDrupalValue(entity.fieldIntroText),
-  fieldLeadership: entity.fieldLeadership[0],
+  fieldLeadership: entity.fieldLeadership.map(i => ({
+    entity: i,
+  })),
```

Comment:
Based on the [liquid template](https://github.com/department-of-veterans-affairs/vets-website/blob/47224bcf0090569cea4066b5702d1034c19c07a9/src/site/layouts/leadership_listing.drupal.liquid#L59-L61) we should iterate all records, however, the transformer is only returning 1 object.
```
   {% for bio in iterator %}
   {% include "src/site/teasers/bio.drupal.liquid" with node = bio.entity %}
   {% endfor %}
```